containers/server-systemd-services/uyuni-server.service

```diff
@@ -0,0 +1,81 @@
+# container-uyuni-server.service
+# autogenerated by Podman 4.3.1
+# Tue Feb 28 17:20:52 CET 2023
+
+[Unit]
+Description=Uyuni server image container service
+Wants=network.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=-/etc/sysconfig/uyuni-server-systemd-services
+Restart=on-failure
+ExecStartPre=/bin/rm \
+	-f %t/uyuni-server.pid %t/%n.ctr-id
+ExecStart=/usr/bin/podman run \
+        --conmon-pidfile %t/uyuni-server.pid \
+	--cidfile=%t/%n.ctr-id \
+	--cgroups=no-conmon \
+	--rm \
+	--sdnotify=conmon \
+	-d \
+	--replace \
+	--tmpfs /run \
+	-p 443:443 \
+	-p 80:80 \
+	-p 4505:4505 \
+	-p 4506:4506 \
+	-p 69:69 \
+	-p 25151:25151 \
+	-p 5432:5432 \
+#TODO these ports will be required in debug mode
+#	-p 8000:8000 \
+#	-p 8001:8001 \
```

Comment:
Would be better to move those to the sysconfig file in the `EXTRA_POD_ARGS` comment.